packages/mask/src/extension/dashboard/index.tsx

```diff
@@ -48,6 +48,8 @@ startPluginDashboard(
             personaSign: Services.Identity.signWithPersona,
             walletSign: Services.Ethereum.personalSign,
             currentPersona: currentPersonaSub,
+            silentSign: Services.Identity.generateSignResult,
```

Comment:
Do you really need this silentSign everwhere? if not, please define it in the SNSContext or WorkerContext

Reply:
You did not do the permission check. Please only allow this on the plugin you want to.